src/components/DeleteCar.jsx

```diff
@@ -0,0 +1,35 @@
+import { useSelector, useDispatch } from "react-redux";
+import { getCars, deleteCar } from "../Redux/cars/carsSlice";
+import { useEffect } from "react";
+
+const DeleteCar = () => {
```

Comment:
- [ ] I can't delete a car that has been reserved:

![image](https://github.com/Zohra-Neda/DriveShare-Front-End/assets/57421823/b374831b-215c-45d5-bf45-7448694b4dbb)

Since there is no way to delete reservations, the car can never be deleted.

Kindly choose at least 1 solution:

1. Add a button to delete reservations.

 #### AND/OR

2. Check why this line is not working properly in the reservation model:

![image](https://github.com/Zohra-Neda/DriveShare-Front-End/assets/57421823/77979f0e-4151-4787-bb14-8d495296973f)

That way, any car will be able to be deleted by the app administration, regardless of reservation.

_tip: My initial guess is that you are missing a "has_many" reservations or "has_one" reservations in the cars model.  I would check if that works first._